client/client-multi/client-java-core/src/test/java/org/kaaproject/kaa/client/persistance/KaaClientPropertiesStateTest.java

```diff
@@ -38,16 +38,19 @@
 
 public class KaaClientPropertiesStateTest {
 
+    private static final String workDir = "work_dir" + System.getProperty("file.separator");
+
     public static KaaClientProperties getProperties() throws IOException {
-        KaaClientProperties props = new KaaClientProperties(CommonsBase64.getInstance());
-        props.setProperty("state.file_location", "state.properties");
-        props.setProperty("keys.public", "key.public");
-        File pub = new File("key.public");
+        KaaClientProperties props = new KaaClientProperties();
+        props.setProperty("kaa.work_dir", "work_dir/");
```

Comment:
Use constants from the KaaClientPropertiesState class instead of hard-coded property names.